src/test/regress/expected/multi_router_planner.out

```diff
@@ -862,6 +898,7 @@ SELECT * FROM (
 ORDER BY id;
 DEBUG:  Creating router plan
 DEBUG:  Plan is router executable
+DETAIL:  distribution column value: 3
```

Comment:
this is a multi-tenant query. We should not be picking arbitrary one here. 
Plus this query gives different results in mac vs linux